changelog/unreleased/enhancement-set-expiration-date-in-three-dots-menu

```diff
@@ -0,0 +1,4 @@
+Enhancement: Moving "set expiration date" function to a three dots menu to help reduce visual load on simple shares in sharing pane
+
+https://github.com/owncloud/web/issues/9493
+https://github.com/owncloud/web/pull/9584
```

Comment:
The title is too long: https://drone.owncloud.com/owncloud/web/38215/4/2. You could just use a shorter one and use this one as description, as our changelog items should always have one.